src/audit/pull_from_queue.py

```diff
@@ -81,11 +81,27 @@ async def pull_from_queue_loop():
     """
     logger.info("Starting to pull from queue...")
     aws_sqs_config = config["QUEUE_CONFIG"]["aws_sqs_config"]
+    # we know the cred is in AWS_CREDENTIALS (see `AuditServiceConfig.validate`)
```

Comment:
Should we consider more logger.info or logger.debug lines here to facilitate some debugging later down the line?

Reply:
as mentioned in the comment, it's already checked in the config validation, so the service would not start if it's misconfigured. when we reach this point there shouldn't be anything to log